src/Jira/Api/Client/CurlClient.php

```diff
@@ -65,7 +61,9 @@ public function sendRequest($method, $url, $data = array(), $endpoint, Authentic
         curl_setopt($curl, CURLOPT_URL, $endpoint . $url);
         curl_setopt($curl, CURLOPT_HEADER, 0);
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
-        curl_setopt($curl, CURLOPT_USERPWD, sprintf("%s:%s", $credential->getId(), $credential->getPassword()));
+        if ($credential instanceof Basic) {
```

Comment:
Please replace with `if (!($credential instanceof Anonymous)) {` (approach from `PHPClient` class) to allow custom authentication ways to be used.